app/views/merchant_invoices/show.html.erb

```diff
@@ -10,7 +10,11 @@
     <h3>Item: <%=invoice_item.item.name%></h3>
     <p>Quantity Ordered: <%=invoice_item.quantity%></p>
     <p>Unit Price: $<%=(invoice_item.unit_price.to_f/100)%></p>
-    <p>Status: <%=invoice_item.status%></p>
+      <div id="status-<%= invoice_item.item_id%>">
+        <%= form_with url: "/merchants/#{@merchant.id}/invoices/#{@invoice.id}/#{invoice_item.id}", method: :patch, local: true do |f| %>
```

Comment:
should we try refactoring this with the form_with :model? I wouldn't mind taking a stab at this to practice forms and advanced routing.

Reply:
I had originally attempted the form_with model, but it wouldn't recognize the `invoice_item`, I ended up needing to interpolate the `invoice_item.status` later in the form to set the default value to what was coming in.

If you can make it work, I'm all for it :D